modules/updating-eus-to-eus-upgrade.adoc

```diff
@@ -0,0 +1,144 @@
+To view the status of all MachineConfigPools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. To pause the MachineConfigPools you wish to skip reboots on, run the following commands:
++
+[NOTE]
+====
+You cannot pause the master pool.
+====
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":true}}'
+$ oc get mcp -o
+----
++
+.Example output
++
+[source,terminal]
+----
+custom-columns=NAME:.metadata.name,PAUSED:.spec.paused
+NAME 	    PAUSED
+master      false
+worker      true
+----
+
+. To change to the `eus-4.10` channel and upgrade to 4.9, run the following commands:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.10
+$ oc adm upgrade --to-latest
+----
++
+.Example output
++
+[source,terminal]
+----
+Updating to latest version 4.9.11
+----
+
+. To ensure the 4.9 upgrades is completed successfully retrieve the cluster version, run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.9.11   True       False         6m29s   Cluster version is 4.9.11
+----
+
+. To upgrade to 4.10, run the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade --to-latest
+----
+
+. To ensure the 4.10 upgrade is completed successfully retrieve the cluster version, run the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME  	  VERSION  AVAILABLE  PROGRESSING   SINCE   STATUS
+version   4.10.1   True       False         6m29s   Cluster version is 4.10.1
+----
+
+. To unpause all previously paused MachineConfigPools, run the following command:
++
+[source,terminal]
+----
+$ oc patch mcp/worker --type merge --patch '{"spec":{"paused":false}}'
+----
++
+[NOTE]
+====
+If pools are not unpaused, the cluster is not permitted to upgrade to any future minors and maintenance tasks such as certificate rotation are inhibited. This puts the cluster at risk for future degradation.
+====
+
+. To verify that your previously paused pools have updated and you cluster completed the upgrade to 4.10, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+Output is trimmed for clarity:
++
+[source,terminal]
+----
+NAME 	   CONFIG                                            UPDATED     UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c    True  	 False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5    True 	 False
+----
```

Comment:
We could give a new mc config here(in contrast the old one before update in step 1).  
```
master   rendered-master-52da4d2760807cb2b96a3402179a9a4c
worker   rendered-worker-4756f60eccae96fb9dcb4c392c69d497
```